src/ontology/cl-edit.owl

```diff
@@ -22870,6 +22874,10 @@ AnnotationAssertion(terms:contributor obo:CL_0009006 <https://orcid.org/0000-000
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0009006 "small intestine enteroendocrine cell")
 AnnotationAssertion(oboInOwl:inSubset obo:CL_0009006 pato:location_grouping)
 AnnotationAssertion(rdfs:label obo:CL_0009006 "enteroendocrine cell of small intestine")
+AnnotationAssertion(Annotation(dc:license "http://creativecommons.org/licenses/by/4.0/") Annotation(rdfs:comment "Figure depicts a cross section of intestinal villi in the small intestine, including the crypts of Lieberkuhn (seen here as ‘crypt’), as well as the layer specific locations of cells that exist within the gastrointestinal tract. Layers depicted here include epithelium (rows of cells bordering villus), submucosa (white space below the epithelium containing networks of neurons and ganglia depicted in pink), muscularis mucosae (blue layer), muscularis propria (referred to here as muscle layer).
+
+ISC: Intestinal Stem Cell 
+TA: Transit Amplifying Cells") foaf:depiction obo:CL_0009006 <https://github.com/obophenotype/cell-ontology/blob/gut_images_2/images/SI%20image.png?raw=true>)
```

Comment:
Should be able to use PURL to resolve:

http://purl.obolibrary.org/obo/cl/images/images/SI%20image.png. 

Note - better to have image path names without special characters, so avoid URL conversion like this `%20`